### PR TITLE
(refactor) markers for alerts render as alerts with info, bar capacity also renders to page

### PR DESCRIPTION
**Pull request by tenhold**

### CODE REVIEW

File: server/routes/bar.js [hunk 1]
```diff
@@ -46,11 +46,21 @@ barRouter.get('/', (req, res) => {
     });
 });
 
+barRouter.get('/currentOcc/:bar', (req, res) => {
+  const { bar } = req.params;
+  Customers_Bars.findAll({
+    where: { id_bar: bar },
+  })
+    .then((data) => {
```
Comment (jstrzesz) [inline server/routes/bar.js:54]: .catch block needed

File: server/routes/cb.js [hunk 2]
```diff
@@ -185,16 +183,16 @@ cbRouter.delete('/checkout', (req, res) => {
 });
 
 cbRouter.get('/list', (req, res) => {
-  const { barId } = req.query
+  const { barId } = req.query;
   Customers_Bars.findAll({ where: { id_bar: barId, checkin: true } })
     .then((list) => {
```
Comment (jstrzesz) [inline server/routes/cb.js:188]: .catch block needed

File: src/components/Customer/Create/BarInfo.jsx [hunk 3]
```diff
@@ -41,6 +41,8 @@ const BarInfo = ({ placeInfo, searchMarker }) => {
   const photo = photos[0].getUrl();
   const classes = useStyles();
 
+  console.log('customerId in barinfo', customerId);
```
Comment (jstrzesz) [inline src/components/Customer/Create/BarInfo.jsx]: remove

File: src/components/Customer/Map/FriendsMarker.jsx [hunk 4]
```diff
@@ -1,8 +1,18 @@
...
 import { Marker, InfoWindow } from '@react-google-maps/api';
+// import { ReportProblem } from '@material-ui/icons';
+import warning from '../../../../images/warning.png';
 
-const FriendsMarker = ({ friendsLocation: { user_name, latitude, longitude } }) => {
+const FriendsMarker = ({ friendsLocation: { user_name, latitude, longitude, report } }) => {
   const [show, setShow] = useState(false);
+  const iconSelect = report ? warning : 'http://maps.google.com/mapfiles/ms/icons/blue.png';
+
+  // const iconSelect = () => {
```
Comment (jstrzesz) [inline src/components/Customer/Map/FriendsMarker.jsx]: remove if not needed

**Review by lschwall — APPROVED** [review]

**Review by Bri-B — APPROVED** [review]

**Review by christophermbooth — APPROVED** [review]

**Pull request merged**